jib-core/src/integration-test/java/com/google/cloud/tools/jib/builder/BuildStepsIntegrationTest.java

```diff
@@ -103,7 +105,7 @@ private static void assertDockerInspect(String imageReference)
     Assert.assertThat(history, CoreMatchers.containsString("bazel build ..."));
   }
 
-  private static final TestJibLogger logger = new TestJibLogger();
+  private static final Logger LOGGER = LoggerFactory.getLogger(BuildStepsIntegrationTest.class);
```

Comment:
It was customary to name it `logger` (lower cases). I heard like the logger may not be immutable.